### ISSUE TITLE
undefined method `random_timezone' for Zonebie

### ISSUE THREAD

**Issue by paveldruzyak** [alindeman/zonebie#15]
Please, don't remove public method from your API in minor updates.
```
[1] pry(main)> Zonebie.random_timezone
NoMethodError: undefined method `random_timezone' for Zonebie:Module
```

**Comment by typeoneerror** [comment]
Definitely a useful public API still. I vote for bringing back the `random_timezone` method.

**Comment by alindeman** [comment]
Yes. This sounds like a mistake. Is one of you willing to make a pull
request?

**Comment by typeoneerror** [comment]
@alindeman sure

**Comment by alindeman** [comment]
I've merged in @typeoneerror's fix in #16 and released Zonebie 0.6.1. Thanks for bringing up this issue.
